bedrock/firefox/templates/firefox/channel/base.html

```diff
@@ -56,7 +56,7 @@ <h2 class="c-sub-navigation-title"></h2>
   </ul>
   {% endcall %}
 
-  <section class="t-newsletter">
+  <section class="mzp-c-emphasis-box t-newsletter">
```

Comment:
Apologies if this was confusing from the issue description. 
I think we should do custom styles here instead of the default emphasis box, similar to what we did for the newsletter in `developer/includes/newsletter.scss`

So in `media/css/firefox/channel.scss` we can do something like:
```
.t-newsletter {
    padding: $layout-md $layout-xs;

    .mzp-l-content {
        border-radius: 6px;
        box-shadow: 0 4px 16px 0 $color-gray-60;
        max-width: $content-sm;
        min-width: 0;
        padding-bottom: $layout-md;
        padding-top: $layout-md;
    }
```

![Screen Shot 2020-01-02 at 8 26 23 AM](https://user-images.githubusercontent.com/30483988/71678307-b5850180-2d39-11ea-90ca-2fece954350b.png)